content/en/docs/tasks/configmap-secret/managing-secret-using-kubectl.md

```diff
@@ -105,61 +111,88 @@ Type:            Opaque
 
 Data
 ====
-password:    12 bytes
+password:    16 bytes
 username:    5 bytes
 ```
 
 The commands `kubectl get` and `kubectl describe` avoid showing the contents
 of a `Secret` by default. This is to protect the `Secret` from being exposed
 accidentally, or from being stored in a terminal log.
 
-## Decoding the Secret  {#decoding-secret}
+### Decode the Secret  {#decoding-secret}
 
-To view the contents of the Secret you created, run the following command:
+1.  View the data stored in the Secret:
 
-```shell
-kubectl get secret db-user-pass -o jsonpath='{.data}'
-```
+    ```shell
```

Comment:
We may always try avoid indenting lines by 4 spaces. Markdown may treat this kind of text as code segments.

So .. Line 124 should be `1. View ...`, with only one space before the 'V' character.

Reply:
I think it's because I use tabs to get even spacing. I'll try to modify all the ones I can catch in my next pass through